src/callbacks_step/save_solution.jl

```diff
@@ -52,25 +64,74 @@ function Base.show(io::IO, ::MIME"text/plain", cb::DiscreteCallback{<:Any, <:Sav
   end
 end
 
+function Base.show(io::IO, ::MIME"text/plain",
+                   cb::DiscreteCallback{<:Any, <:PeriodicCallbackAffect{<:SaveSolutionCallback}})
+  @nospecialize cb # reduce precompilation time
+
+  if get(io, :compact, false)
+    show(io, cb)
+  else
+    save_solution_callback = cb.affect!.affect!
 
-function SaveSolutionCallback(; interval=0,
+    setup = [
+             "dt" => save_solution_callback.interval_or_dt,
+             "solution variables" => save_solution_callback.solution_variables,
+             "save initial solution" => save_solution_callback.save_initial_solution ? "yes" : "no",
+             "save final solution" => save_solution_callback.save_final_solution ? "yes" : "no",
+             "output directory" => abspath(normpath(save_solution_callback.output_directory)),
+            ]
```

Comment:
```suggestion
    if save_solution_callback.interval_or_dt isa Integer
      interval_or_dt = "interval"
    else
      interval_or_dt = "dt"
    end
    
    setup = [
             interval_or_dt => save_solution_callback.interval_or_dt,
             "solution variables" => save_solution_callback.solution_variables,
             "save initial solution" => save_solution_callback.save_initial_solution ? "yes" : "no",
             "save final solution" => save_solution_callback.save_final_solution ? "yes" : "no",
             "output directory" => abspath(normpath(save_solution_callback.output_directory)),
            ]
```
To avoid confusion?

Reply:
I thought this was taken care of already because there are two versions of `Base.show` that dispatches on `PeriodicCallbackAffect` or not. So the code above will only ever trigger in the `dt` case.

Reply:
You're right, my bad.

Reply:
No worries :)